src/main/docs/guide/introduction/whatsNew.adoc

```diff
@@ -19,6 +19,18 @@ The official GraalVM Maven plugin has new GAV coordinates so if you have declare
 
 Please check https://graalvm.github.io/native-build-tools/0.9.7.1/maven-plugin.html[the official documentation] about how to customize the plugin.
 
+=== Gradle plugin 3.0.0
+
+A new major version of the Gradle plugin has been released, including internal changes to use Gradle's lazy configuration APIs and make bugfixing simpler.
+In the process, https://micronaut-projects.github.io/micronaut-gradle-plugin/snapshot/[documentation] has been rewritten.
```

Comment:
This should point to the `latest` docs. Also it seems only a milestone is still out there? 3.0.0 needs releasing